compiler/modules/types/core.bal

```diff
@@ -306,6 +302,13 @@ public type ComplexSemType readonly & record {|
     ProperSubtypeData[] subtypeDataList;
 |};
 
+// This is to represent a SemType belonging to cell basic type
+public type MemberSemType readonly & record {|
```

Comment:
I thought we decided that we were weren't going to do this and would instead just do

```
type MemberSemType SemType;
```

Then use MemberSemType whenever a SemType satisfied the constraint that if it contains anything beginning to the Cell basic type, then it is a single Node cell Bdd.